BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -40,6 +40,20 @@ public async void LandlordList_WhenCalled_CallsGetLandlordListAndReturnsLandlord
         A.CallTo(() => AdminService.GetLandlordDisplayList("")).MustHaveHappened();
         result!.ViewData.Model.Should().BeOfType<LandlordListModel?>();
     }
+    
+    [Fact]
+    public async void TenantList_WhenCalled_CallsGetTenantListAndReturnsTenantListView()

```

Comment:
This doesn't check that it "CallsGetTenantList"

Reply:
I'll fix that in BNH-116, I want to get this branch merged so that it's easier to work on everything tenant-related.